Dockerfile

```diff
@@ -208,14 +224,25 @@ COPY --chown=${UID} . ${RAY_JL_PROJECT}/
 
 # Restore content from previous build directory
 RUN rm -rf ${BUILD_PROJECT} && \
-    ln -s ${BUILD_ROOT} ${BUILD_PROJECT}
+    mv ${BUILD_ROOT} ${BUILD_PROJECT}
```

Comment:
Good question. We now move the `BUILD_ROOT` into the `BUILD_PROJECT` as this will include the content in the `JULIA_USER_DEPOT`. Using a symlink would mean when we do `COPY --from=build-ray-jl ${JULIA_USER_DEPOT} ${JULIA_USER_DEPOT}` the `BUILD_ROOT` would not also be copied.